openmc/mgxs_library.py

```diff
@@ -296,11 +296,11 @@ def xs_shapes(self):
             self._xs_shapes["[G][G']"] = (self.energy_groups.num_groups,
                                           self.energy_groups.num_groups)
             self._xs_shapes["[DG]"] = (self.num_delayed_groups,)
-            self._xs_shapes["[G][DG]"] = (self.energy_groups.num_groups,
+            self._xs_shapes["[DG][G]"] = (self.energy_groups.num_groups,
```

Comment:
The left hand side (`[DG][G]`) is inconsistent with the right hand side for these changes.